src/Nox.Generator/Application/Queries/TranslationsByIdQuery.template.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Generated
+
+#nullable enable
+
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+using Nox.Application.Commands;
+
+using {{codeGeneratorState.ApplicationNameSpace}}.Dto;
+using {{codeGeneratorState.PersistenceNameSpace}};
+
+namespace {{codeGeneratorState.ApplicationNameSpace}}.Queries;
+
+public record Get{{entity.Name }}TranslationsByIdQuery({{primaryKeys}}, System.String {{codeGeneratorState.LocalizationCultureField}}) : IRequest <IQueryable<{{entity.Name}}LocalizedDto>>;
```

Comment:
can use the class name property injected in Scriban
```suggestion
public record {{className}}({{primaryKeys}}, System.String {{codeGeneratorState.LocalizationCultureField}}) : IRequest <IQueryable<{{entity.Name}}LocalizedDto>>;
```

Reply:
Desired changes has been applied